content/en/blog/2022-03-11-flagger-is-adding-gateway-api/index.md

```diff
@@ -15,6 +15,21 @@ part of the [1.19.0 release](https://github.com/fluxcd/flagger/releases/tag/v1.1
 Read here about why this is a significant development in Flagger and how
 you can make use of it.
 
+## What is Flagger?
+
+Flagger is a progressive delivery tool that automates the release process for applications running on Kubernetes.
```

Comment:
Maybe add a link to the Flagger website?

Reply:
I added below a link to the repo.